company.c

```diff
@@ -0,0 +1,47 @@
+/*Priyanka Musti*/
+/*Write a program company.c that implements an employee structure containing a name, position, and a salary. Create an array of 5 employees in your main function, each with a different name and salary. Write a function output() that takes an employee array and prints out the array of employees*/
+#include<stdio.h>
+#include<string.h>
+#include<math.h>
+#include<stdlib.h>
+
+struct employee
+{
+	char name [100];
+	char position[100];
+	long int salary;
+	
+};
+int main()
+{
+	int x;
+	struct employee ploy[5];
+	strcpy(ploy[0].name, "Taylor Swift");
+	strcpy(ploy[0].position, "Journalist");
+	ploy[0].salary= 45000;
+
+
+	strcpy(ploy[1].name, "Selena Gomez");
+	strcpy(ploy[1].position, "Secretary");
+	ploy[1].salary= 25000;
```

Comment:
these should have been printed out with two decimal points! Also, again, include comments! Other than that, very nicely done.